zendesk_apps_tools.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |s|
   s.required_ruby_version = '>= 2.3'
   s.required_rubygems_version = '>= 1.3.6'
 
-  s.add_runtime_dependency 'thor',        '~> 0.19.4'
+  s.add_runtime_dependency 'thor',        '~> 1.2.1'
```

Comment:
this is a major version change which I guess require us to test it before release it out :)